adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/CommandSubscriptionsManager.java

```diff
@@ -70,18 +70,25 @@ public CommandSubscriptionsManager(final Vertx vertx, final T config) {
      * Invoked when a device sends an MQTT <em>PUBACK</em> packet.
      *
      * @param msgId The message/packet id of the command published with QoS 1.
+     * @return {@code true} if a pending command request for the PUBACK was found.
      * @throws NullPointerException if msgId is {@code null}.
      */
-    public void handlePubAck(final Integer msgId) {
+    public boolean handlePubAck(final Integer msgId) {
```

Comment:
I see that the return value is not being used anywhere. Is it here in consideration with the upcoming PRs?

Reply:
Yes, right. The PUBACK could have been sent for a published command *or* a published message on the new error topic for #2064. So, the PUBACK isn't necessarily only handled in the CommandSubscriptionsManager any more.